src/components/pages/room.yml

```diff
@@ -44,3 +44,18 @@ room:
   pendingInvitation:
     en: Pending invitation
     de: Ausstehende Einladung
+  messageBoardSectionHeadline:
+    en: Message board
+    de: Message board
```

Comment:
Better: "Message-Board"

Reply:
In German of course it does 😅 I had just written both in English. Changed, thanks!